src/commonMain/kotlin/operations/data/Var.kt

```diff
@@ -50,11 +51,11 @@ internal object Var : LogicExpression {
         && expression.size > 1
 
     private fun getRecursive(indexes: List<String>, data: List<Any?>): Any? = indexes.firstOrNull()?.apply {
-        val indexedData = data.getOrNull(intValue)
+        val indexedData = data.getOrNull(intOrZero)
```

Comment:
Can we consider not leaving those extensions without 'this'? While reading in IntelliJ, it is not that bad, bad looking at code outside of IDE makes this confusing where this field is coming from

Reply:
Seems like pure personal feeling. For me its quite obvious when I see apply operator. I agree on that when function is more like 20 lines long but this is not the case :D